### ISSUE TITLE
warnings from compiler about javascript versions

### ISSUE THREAD

**Issue by thedavidmeister** [tonsky/datascript#236]
I'm seeing this in newer versions of cljs:

```
datascript/parser.js:7241: WARNING - Keywords and reserved words are not allowed as unquoted property names in older versions of JavaScript. If you are targeting newer versions of JavaScript, set the appropriate language_in option.
```

**Comment by thedavidmeister** [comment]
the generated code it is complaining about looks like this:

```
datascript.parser.Query.prototype.cljs$core$IEquiv$_equiv$arity$2 = (function (this39772,other39773){
var self__ = this;
var this39772__$1 = this;
return (!((other39773 == null))) && ((this39772__$1.constructor === other39773.constructor)) && (cljs.core._EQ_.cljs$core$IFn$_invoke$arity$2(this39772__$1.find,other39773.find)) && (cljs.core._EQ_.cljs$core$IFn$_invoke$arity$2(this39772__$1.with,other39773.with)) && (cljs.core._EQ_.cljs$core$IFn$_invoke$arity$2(this39772__$1.in,other39773.in)) && (cljs.core._EQ_.cljs$core$IFn$_invoke$arity$2(this39772__$1.where,other39773.where)) && (cljs.core._EQ_.cljs$core$IFn$_invoke$arity$2(this39772__$1.__extmap,other39773.__extmap));
});
```

**Comment by tonsky** [comment]
I fixed that in https://github.com/tonsky/datascript/commit/898938250fd1bce7757c1a265e6633dea7e7d3fa. Just released 0.16.2 with that fix

**Comment by thedavidmeister** [comment]
great, thanks!
